docs/v1.X/docs/references/alerts.md

```diff
@@ -4,5 +4,5 @@ An `Alert` is the runtime representation of a bucket overflow.
 
 The representation of the object can be found here : 
 
-[Alert object documentation](https://pkg.go.dev/github.com/crowdsecurity/crowdsec/pkg/types#RuntimeAlert)
+[Alert object documentation](https://pkg.go.dev/github.com/crowdsecurity/crowdsec/pkg/models#Alert)
```

Comment:
no for this one I reall meant the `types#RuntimeAlert` as it's going to be the runtime one